### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by flutterexcel**

### CODE REVIEW

File: assets/config/prod.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
     "baseUrl": "https://apistaginghr.excellencetechnologies.in",
-    "apiUrl" : "https://apistaginghr.excellencetechnologies.in/attendance/API_HR/api.php"
+    "apiUrl" : "https://apistaginghr.excellencetechnologies.in/attendance/API_HR/api.php",
+    "salUrl" : "https://apistaginghr.excellencetechnologies.in/attendance/sal_info/api.php"
```
Comment (manishiitg) [inline assets/config/prod.json:4]: this is not correct wayt to manage this.

https://medium.com/@matt.goodson.business/separating-build-environment-configurations-in-flutter-with-firebase-doing-it-the-right-way-c72c3ad3621f

read this or similar articles

File: lib/services/inventory/add_inventory_audit.dart [hunk 2]
```diff
@@ -26,7 +26,7 @@ class AddInventoryAudit {
         .then((dynamic res) async {
       if (res["error"] >= 1) throw new Exception(res["data"]["message"]);
       if (res["error"] == 0) {
-        StorageUtil.setUserToken(res["data"]["token"]);
+        //StorageUtil.setUserToken(res["data"]["token"]);
```
Comment (manishiitg) [inline lib/services/inventory/add_inventory_audit.dart:29]: why this removed?

Comment (flutterexcel) [reply lib/services/inventory/add_inventory_audit.dart:29]: no need to save token again.

File: assets/config/prod.json (hunk 1 above)
Comment (flutterexcel) [reply assets/config/prod.json:4]: it's fixed

**Pull request merged**